extensions/vertx-http/runtime/src/main/java/io/quarkus/devui/runtime/config/AuthJsonRPCService.java

```diff
@@ -0,0 +1,124 @@
+package io.quarkus.devui.runtime.config;
+
+import static io.quarkus.devui.runtime.config.PermissionSet.QUARKUS_NULL_VALUE;
+
+import java.util.*;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+
+import org.jboss.logging.Logger;
+
+import io.quarkus.dev.console.DevConsoleManager;
+import io.quarkus.devui.runtime.comms.JsonRpcMessage;
+import io.quarkus.devui.runtime.comms.MessageType;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+@ApplicationScoped
+public class AuthJsonRPCService {
+    private static final Logger LOG = Logger.getLogger(AuthJsonRPCService.class.getName());
+    private final List<String> METHODS = List.of("GET", "POST", "PUT", "DELETE", "PATCH", "OPTIONS", "HEAD", "CONNECT",
+            "TRACE"); // do not change order
+    private final List<String> AUTH_MECHANISM = List.of(QUARKUS_NULL_VALUE, "basic", "bearer", "form"); // do not change order
```

Comment:
there is much more of auth mechanisms and having hardcoded string list of 4 is too little, I wonder if it would be possible to collect these names from `io.quarkus.smallrye.jwt.runtime.auth.JWTAuthMechanism#getCredentialTransport` => `io.quarkus.vertx.http.runtime.security.HttpCredentialTransport#getAuthenticationScheme` that of all `Instance<HttpAuthenticationMechanism> httpAuthenticationMechanism`